local/config/finna/config.ini.sample

```diff
@@ -1197,6 +1197,10 @@ hide_holdings[] = "World Wide Web"
 ;                            always collapse journals
 ;holdings_details = "expand-first,collapse-journals"
 
+; Uncomment to prevent deduplicated holdings from being fetched immediately.
+; Will display choose library placeholder in the deduplicated select elements.
```

Comment:
Laita `...display a "Choose library" placeholder in the record source select elements.`